README.md

```diff
@@ -51,7 +51,7 @@ Next, lets configure the development server. To keep things simple we can put th
 var hypernova = require('hypernova/server');
 
 hypernova({
-  devMode: true,
+  clustering: false,
```

Comment:
Speaking of :) My take on it, that clustering shouldn't be default option, but rather something you opt-in to.
If it's ran in production, it's ran using all sets of configs and environment variables, which leave no thing to default,
and imposing clustering where default matters - in dev mode will make things more confusing than helpful.

So how about keeping it `clustering`, making default `false` and bumping major version? :)